docs/apis-tools/camunda-api-rest/specifications/correlate-a-message.api.mdx

```diff
@@ -1,8 +1,8 @@
 ---
 id: correlate-a-message
-title: "Correlate a Message"
-description: "Publishes a Message and correlates it to a subscription. If correlation is successful it"
-sidebar_label: "Correlate a Message"
+title: "Correlate a message"
+description: "Publishes a message and correlates it to a subscription. If correlation is successful it"
+sidebar_label: "Correlate a message"
```

Comment:
@christinaausley because these files are all generated from the OpenAPI spec, we should look into making these changes at the source. Otherwise, every time we regenerate the spec, the non-preferred text is going to be reintroduced.

@remcowesterhoud can you point us at a location to make these kinds of changes in the original spec?

Reply:
The original spec lives in the Zeebe repository. But as you can see [there we do have a full description](https://github.com/camunda/camunda/blob/main/zeebe/gateway-protocol/src/main/proto/rest-api.yaml#L747-L749). Since I've copied that file to the docs we also have the[ full description here](https://github.com/camunda/camunda-docs/pull/4166/files#diff-ae15829edce6a9cf4c58547d264290393a9c3fb5789b3db7d30aa3cbda0cc704R747-R749).

I suppose the generation cannot handle multiline strings? 🤔

Reply:
I wonder if this is a big issue as it's just the page description, which as far as I know is not listed anywhere. On the page itself all looks good to me.

Reply:
> I suppose the generation cannot handle multiline strings? 🤔

That's very possible! I'll take a look and see what I learn.

Reply:
@pepopowitz I've updated the spec accordingly 👍

Reply:
Welllll this is a significantly longer and more nuanced response than I expected to be writing, but here we go. 

## My research findings: 

The OpenAPI generator that drives our API doc generation uses the `description` property on an endpoint in the OpenAPI spec for two things: 
   1. The generated page's body. The OpenAPI generator takes all lines of the `description` property for this bit, as we can see [when viewing the body of the page associated with this specific file](https://preview.docs.camunda.cloud/pr-4166/docs/next/apis-tools/camunda-api-rest/specifications/correlate-a-message/).
   2. This `description` property in the generated markdown's frontmatter. Docusaurus turns that frontmatter property into the `<meta name="description">` tag on the emitted HTML page. The OpenAPI generator takes only the first line of the property for this -- I think this is a reasonable limitation, and we have other examples where it is helpful, because the spec's `description` includes notes that we probably don't want in the page's meta description. (Example: https://github.com/camunda/camunda-docs/blob/d343afc17e2ed92c85103a40a5687f17a2537dec/api/camunda/camunda-openapi.yaml#L544-L551)

## My proposal: 

We align our `description` properties to this limitation by making the first line a complete sentence and high-level summary of the endpoint. Additional information can come on subsequent lines, but the first line itself should make sense in isolation of the rest of the property. 

### Where do we make those changes?

Ideally, we would make those changes at the source (the spec in the API project itself), so that we don't introduce regressions every time we re-generate these docs. 

However, there may be linting involved in that project that is handling the line breaks automatically. If that's the case, we could make the changes in this project only, and either:
   1. Add a function to [this API's generation strategy](https://github.com/camunda/camunda-docs/issues/3568) that detects the multi-line descriptions and inserts an end of line after the first sentence, or
   2. Adjust it manually in the OpenAPI spec in this PR, and then count on us noticing an unexpected diff when we re-generate the docs, and fix it.

2 is obviously ripe for human error, so 1 would be my preference....however I'd open a separate issue to take care of that, as I don't think it's urgent. 

## So what does this PR need to do to move forward?

If my proposal seems reasonable: 

1. We adjust multi-line `description` properties in this API's spec within this project, so that there is a line break after the first sentence. Assignee: if anyone wants to do this, go for it, otherwise I'll take care of it tomorrow.
2. We follow up this PR with a change to the OpenAPI spec in the API's project, to align with this change & Christina's technical review changes, to avoid regressions at next generation. Assignee: me, and @christinaausley if she wants to pair on it.
3. If the line-breaks I'm proposing are impossible or rejected in that project, I'll create an issue to add a step to this API's generation strategy to ensure multi-line descriptions start with one complete sentence. Assignee: me.

If people disagree with my proposal, we'll figure something else out!

Reply:
FYI, I updated the description property in question, as described above, in a new commit on this branch. I also re-generated the docs from the spec.

Reply:
Awesome, thank you @pepopowitz! Does this mean this is good to merge now?

Reply:
I'm good with it if @christinaausley is!

Reply:
Ship itttttt 🚢